.github/workflows/formatting-tests.yaml

```diff
@@ -27,7 +27,6 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8 pytest
-        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
 
     - name: Update black
       if: ${{ matrix.python-version == 3.9 }}
```

Comment:
these tests seem to be running for multiple python versions right? which seems unnecessary